gitops/addons/bootstrap/aws/addons-aws-oss-external-secrets-appset.yaml

```diff
@@ -55,8 +55,6 @@ spec:
             values: |
               serviceAccount:
                 name: {{metadata.annotations.external_secrets_service_account}}
-                annotations:
```

Comment:
Why you remove the IRSA annotations ?

Reply:
Using Pod Identity for the external secret so the IRSA annotation is not needed there

Reply:
Clear thanks